.gitattributes

```diff
@@ -0,0 +1,2 @@
+ __init__.py export-subst
+setup.py export-subst
```

Comment:
Newlines here and in .gitignore would be nice, but won't let it hold up the PR.

Reply:
Addressed in 3efb09f